extensions/wikia/Email/Controller/FandomCreator/MagicWordWrapper.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Email\Controller\FandomCreator;
+
+
+use Wikia\Util\GlobalStateWrapper;
+
+class MagicWordWrapper {
+	private $siteName;
+
+	private $siteUrl;
+
+	public function __construct( string $siteName, string $siteUrl ) {
+		$this->siteName = $siteName;
+		$this->siteUrl = $siteUrl;
+	}
+
+	public function wrap( callable $func, array $additionalOverrides = [] ) {
+		return ( new GlobalStateWrapper( array_merge( [
+				'wgSitename' => $this->siteName, // {{SITENAME}}
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)